network/ieee802_11_prism.ksy

```diff
@@ -0,0 +1,66 @@
+meta:
+  id: ieee802_11_prism
+  title: prism II headers for 802.11 capture
+  license: Unlicense
+  endian: le
+  encoding: ascii
+
+doc-ref:
+  - https://www.tcpdump.org/linktypes/LINKTYPE_IEEE802_11_PRISM.html
+
+seq:
+  - id: message_code
+    type: u4
+  - id: size
+    type: u4
+  - id: device_name
+    type: strz
+    size: 16
+  - id: fields
+    type: fields
+    size: fields_size
+  - id: body
+    size-eos: true
+instances:
+  header_size:
+    value: 4 + 4 + 16
+  fields_size:
+    value: size - header_size
+types:
+  fields:
+    seq:
+      - id: fields
+        type: field
+        repeat: eos
+    types:
+      field:
+        seq:
+          - id: flags
+            type: u1
+          - id: type
+            type: u1
+            enum: type
+          - id: unknown
+            type: u2
+            doc: usually zero
+          - id: is_supplied
+            type: u2
+          - id: size
+            type: u2
+          - id: value
+            type:
+              switch-on: type
+              cases:
+                _: u4
```

Comment:
Please don't use type switching with the else case only, this situation has been broken in some languages until recently (fixed in commits https://github.com/kaitai-io/kaitai_struct_compiler/search?q=else+only+switch&type=Commits). Because your spec doesn't have any other 0.9 only features, I suggest to eliminate this small detail to work nicely even with 0.8.

```suggestion
            type: u4
```